content/docs/command-reference/exp/init.md

```diff
@@ -0,0 +1,137 @@
+# exp init
+
+Initializes experiments.
+
+## Synopsis
+
+```usage
+usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
+                    [--explicit] [--name NAME] [--code CODE]
+                    [--data DATA] [--models MODELS] [--params PARAMS]
+                    [--metrics METRICS] [--plots PLOTS] [--live LIVE]
+                    [--type {default,live}]
+                    [command]
+```
+
+## Description
+
+`dvc exp init` helps you quickly get started with experiments, with the command
+to run and tracks data, params, source code, models, metrics and plots by
+default.
+
+It reduces boilerplate for initializing [pipeline](/doc/command-reference/dag)
+stages in a `dvc.yaml` file by assuming sane defaults about the location of your
```

Comment:
And `dvc exp run`. And maybe `params` and `metrics` for that matter.

Reply:
`exp run` should be linked in the next line. `params` and `metrics` should be easy. I'm having trouble adding `stage add` here.

Reply:
hmm, maybe we can bring `stage add` by mentioning why `exp init` exists instead?

Reply:
> would be good to link to stage add

FYI in #3071 I changed that link for `dvc.yaml` because we want to encourage people to write that manually. And added a link to `stage add` from there.